examples/oneview_enclosure_facts.yml

```diff
@@ -45,12 +64,21 @@
 
     - debug: var=enclosures
 
+    # This block is compatible with C7000 Enclosures
+    - name: Gather facts about an Enclosure by name with script
+      oneview_enclosure_facts:
+        config: "{{ config }}"
+        name: '{{ name }}'
+        options:
+          - script
+      when: variant == 'C7000' and currentVersion == '300'
+      delegate_to: localhost 
+
     - name: Gather facts about an Enclosure by name with options
       oneview_enclosure_facts:
         config: "{{ config }}"
         name: '{{ name }}'
         options:
-          - script                       # optional
```

Comment:
did you delete this because that endpoint is not supported any more..?

Reply:
I removed it here and placed above just to differentiate between variants.

Reply:
ok.. i didn't notice that